tests/mock_image_host.h

```diff
@@ -110,17 +111,9 @@ class MockImageHost : public VMImageHost
     VMImageInfo mock_snapcraft_image_info{
         {snapcraft_alias},       "Ubuntu", "core20", snapcraft_release_info, true, image.url(), snapcraft_image_id, "",
         snapcraft_image_version, 1,        true};
-    VMImageInfo mock_custom_image_info{{custom_alias},
-                                       "Ubuntu",
-                                       "Custom Core",
-                                       custom_release_info,
-                                       true,
-                                       image.url(),
-                                       custom_image_id,
-                                       "",
-                                       custom_image_version,
-                                       1,
-                                       false};
+    VMImageInfo mock_custom_image_info{
+        {custom_alias},       "Ubuntu", "Custom Core", custom_release_info, true, image.url(), custom_image_id, "",
+        custom_image_version, 1,        false};
```

Comment:
Ugh, another `clang-format` change it appears :slightly_frowning_face:

Reply:
This one can be fixed.

Reply:
Thanks!